tests/test_grading.py

```diff
@@ -92,9 +92,9 @@ def test_course_submissions_optimized(self):
             print("Running with {}".format(db.engine.name))
             query = self.assignment.mysql_course_submissions_query()
             data = [d for d in query]
-            backups_from_query = [d[6] for d in data if d[6]]
+            backups_from_query = [d[7] for d in data if d[7]]
```

Comment:
👍 

I think I was hoping to test that the mysql query behaves exactly like the slow query in terms of what backups it obtains - but that's done elsewhere and the code above didn't really test that anyway.